src/app/project/template.html

```diff
@@ -14,312 +14,352 @@
-          mat-flat-button
-          type="button"
-          fxLayoutAlign="center center"
-          [disabled]="isProjectCreationRestricted()"
-          (click)="addProject()">
-    <i class="km-icon-mask km-icon-add"></i>
-    <span>Add Project</span>
-  </button>
+<div *ngIf="!isInitializing && !isProjectsExists else showProjectView"
+     fxLayout="column"
+     fxLayoutAlign="center center"
+     class="placeholder">
+  <div fxLayout="column"
+       fxLayoutAlign="center center">
+    <i class="km-icon-mask km-icon-projects-card i-40"></i>
+    <h2>Get Started</h2>
+    <div fxLayout="column"
+         fxLayoutAlign="center center"
+         class="placeholder-text">
+      <p>Dive right into your first project.</p>
+      <a href="https://docs.kubermatic.com/kubermatic/master/tutorials-howtos/project-and-cluster-management/"
+         target="_blank"
+         rel="noreferrer noopener">
+        Learn more in our documentation >
+      </a>
+    </div>
+    <br />
+    <button mat-flat-button
+            fxLayoutAlign="center center"
+            type="button"
+            [disabled]="isProjectCreationRestricted()"
+            (click)="addProject()">
+      <i class="km-icon-mask km-icon-add"></i>
+      <span>Create Project</span>
+    </button>
+  </div>
 </div>
 
-<!-- Table View -->
-<div *ngIf="!showCards">
-  <mat-card class="km-project-view">
-    <mat-card-content>
-      <table class="km-table"
-             mat-table
-             matSort
-             [dataSource]="dataSource"
-             [trackBy]="projectTrackBy.bind(this)">
-        <ng-container matColumnDef="status">
-          <th mat-header-cell
-              *matHeaderCellDef
-              class="km-header-cell"></th>
-          <td mat-cell
-              *matCellDef="let element">
-            <i [matTooltip]="element.status"
-               [ngClass]="getStatusIcon(element)"
-               class="km-health-state"></i>
-          </td>
-        </ng-container>
+<div class="km-row"
+     *ngIf="isInitializing">
+  <mat-spinner color="accent"
+               class="projects-spinner"
+               [diameter]="25"></mat-spinner>
+</div>
 
-        <ng-container matColumnDef="name">
-          <th mat-header-cell
-              *matHeaderCellDef
-              class="km-header-cell p-20"
-              mat-sort-header="name">Name
-          </th>
-          <td mat-cell
-              *matCellDef="let element"
-              [attr.id]="'km-project-name-' + element.name">{{element.name}}</td>
-        </ng-container>
 
-        <ng-container matColumnDef="labels">
-          <th mat-header-cell
-              *matHeaderCellDef
-              class="km-header-cell p-20">Labels
-          </th>
-          <td mat-cell
-              *matCellDef="let element">
-            <km-labels [labels]="element.labels"
-                       [limit]="2"></km-labels>
-          </td>
-        </ng-container>
+<ng-template #showProjectView>
```

Comment:
Project view should be hidden when spinner is being displayed.

Reply:
Update gif